docs/source/index.rst

```diff
@@ -40,7 +42,7 @@ Glossary
 This is a glossary of terms used throughout this documentation.
 
 * **Context** is the datastore.
-* **Subflow** is a flow/loop task which is a subtask of another flow.
+* **Subflow** is a flow/loop task which is a subtask ASK_MICKY(is just a task of another flow right? dont think we need subtask, makes it sound like a different abstraction) of another flow.
```

Comment:
make clear that a subtask is the same as a task, but exists on another task - this is true of all tasks that aren't the root flow